test/karma.conf.js

```diff
@@ -120,5 +87,72 @@ module.exports = function(config) {
         { type: 'text-summary' }
       ]
     }
-  });
+  };
+
+  if (process.env.TRAVIS) {
+    settings.browserify.transform.push('browserify-istanbul');
+
+    if (process.env.SAUCE_ACCESS_KEY) {
+      settings.browsers = [
+        'chrome_sl',
+        'firefox_sl',
+        'safari_sl',
+        'ie_sl'
+      ];
+    } else {
+      settings.browsers = ['Firefox'];
+    }
+  }
+
+  config.set(settings);
 };
+
+function getCustomLaunchers(){
+  return {
+    chrome_sl: {
+      singleRun: true,
+      base: 'SauceLabs',
+      browserName: 'chrome',
+      platform: 'Windows 8.1',
+      version: '34'
+    },
+
+    firefox_sl: {
+      singleRun: true,
+      base: 'SauceLabs',
+      browserName: 'firefox',
+      platform: 'Linux',
+      version: '29'
+    },
+
+    safari_sl: {
+      singleRun: true,
+      base: 'SauceLabs',
+      browserName: 'safari',
+      platform: 'OS X 10.8'
+    },
+
+    ipad_sl: {
```

Comment:
Yeah, it is disabled now because it's not being added to the browsers list in the `process.env.SAUCE_ACCESS_KEY` conditional. The custom launcher definition here doesn't do anything unless you use it.